app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    problems_count = 0
```

Comment:
No need to count unvaccinated friends, because one it's enough to return "All friends should be vaccinated"